_maps/RandomRuins/SpaceRuins/oldstation.dmm

```diff
@@ -8486,6 +8485,20 @@
 /obj/machinery/duct,
 /turf/open/floor/iron/white,
 /area/ruin/space/ancientstation/charlie/hall)
+"SQ" = (
+/obj/effect/decal/cleanable/shreds,
+/obj/structure/alien/weeds,
+/obj/structure/closet/crate/secure/science{
+	req_access = list("away_science")
+	},
+/obj/item/relic,
+/obj/item/transfer_valve,
+/obj/item/raw_anomaly_core/bluespace,
```

Comment:
You probably don't need to have two anomaly cores. I think the one random one is probably fine, as a treat.

Reply:
It's just that Bluespace has the most utility - you will get the core when you're done fighting, so Tesla armor will be useless without enemies here. With bluespace you can make a bag, or a teleporting modsuit module. To refine, they need to actually get the plasma canister at the other end of the map, and bother with thermomachines. Very few will bother with this, so I think it's fine to have one useful core and one random for fun.

Reply:
I think one or the other would be fine, looking at the random one as my preference. You'd be surprised at the lengths that powergamers would go, especially with static loot spawns.

Reply:
I'm leaning towards bluespace because at least it's a guaranteed useful thing, and the random one is just a fluff. It is good to have two so that you won't build refinery just for one core. And it's a fun luck aspect. Also if the random one ends up being bluespace too, you can create a singularity on oldstation. But you need to clean up the xeno nest and unlock the crate to see if it's even possible. Is it really that bad to keep both?